src/ui/public/vis/map/kibana_map.js

```diff
@@ -483,6 +485,47 @@ export class KibanaMap extends EventEmitter {
     this._updateLegend();
   }
 
+  _addMaxZoomMessage = (() => {
+    const maxZoomToast = {
+      title: 'Max Zoom Warning',
+      text: (
+        <div>
+          <p>
+            You&apos;ve reached the maximum number of zoom levels. To zoom
```

Comment:
Can we [localize](https://github.com/elastic/kibana/tree/master/packages/kbn-i18n) this message?